fan-xiaomi.js

```diff
@@ -1048,7 +1048,7 @@ LED
         // Natural mode
         activeElement = fanboxa.querySelector('.var-natural')
         if (this.supportedAttributes.naturalSpeed) {
-            if (preset_mode === 'Nature') {
+            if (preset_mode === 'Nature' || preset_mode === 'nature') {
```

Comment:
Should we guard against casing differences in general? That would prevent issues if a different fan reported 'NATURE' etc.
```suggestion
            if (preset_mode.toLowerCase() === 'nature') {
```